airbyte-integrations/connectors/source-mssql/src/test/java/io/airbyte/integrations/source/mssql/CdcStateCompressionTest.java

```diff
@@ -182,11 +197,19 @@ public void testCompressedSchemaHistory() throws Exception {
       assertEquals("1", record.getData().get("id").toString());
     }
 
+    LOGGER.info("inserting new data into test tables");
+    List<Callable<MsSQLTestDatabase>> waitForCdcTasks = new ArrayList<>();
     // Insert a bunch of records (1 per table, again).
     for (int i = 0; i < TEST_TABLES; i++) {
-      testdb.with("INSERT %s.test_table_%d DEFAULT VALUES;", TEST_SCHEMA, i);
+      String tableName = "test_table_%d".formatted(i);
+      String cdcInstanceName = "capture_instance_%d_%d".formatted(i, 2);
+      testdb.with("INSERT %s.%s DEFAULT VALUES;", TEST_SCHEMA, tableName);
+      waitForCdcTasks.add(() -> testdb.waitForCdcRecords(TEST_SCHEMA, tableName, cdcInstanceName, 1));
```

Comment:
we were missing this wait, and instead relying upon the 1min initial_waiting_seconds